README.md

```diff
@@ -61,7 +61,7 @@ You can change the HTML/CSS layout if you need it.
 ## Deploy and Pull Request
 
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_2048_game/)
+    - [DEMO LINK](https://ElisabethPO.github.io/js_2048_game/)
```

Comment:
Add this demo link to the PR description 
<img width="964" alt="image" src="https://github.com/user-attachments/assets/eec20db1-cb45-48d0-b19c-73528ae75787" />